### DISCUSSION TITLE
Syntax highlighting for JSDOC comments

### DISCUSSION THREAD

**Discussion by theBGuy** [community/community#51966] [Mobile]
Is there a way to enable more syntax highlighting?

**Comment by iskae** [comment]
Hello @theBGuy 👋 

Unfortunately we do not highlight on Mobile on our own and we are not able to add support for more languages 😅 

For more info on the topic you can refer to [our docs](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) 😊 

Sorry for not being able to help further 🙇

**Comment by Polygons1** [comment]
desktop too
